src/components/ebay-calendar/index.marko

```diff
@@ -0,0 +1,110 @@
+// @ts-check
+import {toISO} from "./component"
+
+$ const numMonths = input.numMonths || 1;
+$ const monthDates = [...Array(numMonths)].map((_, i) => component.getMonthDate(state.offset + i))
+
+<div.calendar>
+    <if(input.navigable)>
+        <div.calendar__header>
+            <div.calendar__header--inner>
+                <ebay-icon-button
+                    transparent size="small"
+                    disabled=(state.disableBefore && component.getFirstVisibleISO() <= state.disableBefore)
+                    aria-label=`Show ${component.monthTitle(component.getMonthDate(state.offset - 1))}`
```

Comment:
This needs to be configurable for localization. Maybe have it as `a11yPrevMonthText`

Reply:
No. We need to allow the users to pass in full text
Generally this text we have is not useful. Think of other languages, they might have the description done after the number or before. It might have to be "march show" or something. 
So there needs to be flexibility here. 
We can provide the basic template but allow it to be overriden. We should also make sure all the data they need to do so is emitted in the events that update it
See carousel also for this. To see how we update the current page etc.